nixos/modules/i18n/input-method/default.nix

```diff
@@ -66,6 +66,8 @@ in
 
   meta = {
     maintainers = with lib.maintainers; [ ericsagnes ];
+    # Don't edit the docbook xml directly, edit the md and generate it:
+    # `pandoc default.md -t docbook --top-level-division=chapter --extract-media=media -f markdown-smart --lua-filter ../../../../doc/build-aux/pandoc-filters/myst-reader/roles.lua --lua-filter ../../../../doc/build-aux/pandoc-filters/docbook-writer/rst-roles.lua > default.xml`
```

Comment:
Maybe we could update `nixos/doc/manual/md-to-db.sh` to support it and have single command for all pages.

Reply:
Yes, that is what I would prefer.